pygpudrive/env/config.py

```diff
@@ -103,3 +100,20 @@ class RenderConfig:
 
     def __str__(self):
         return f"RenderMode: {self.render_mode.value}, ViewOption: {self.view_option.value}, Resolution: {self.resolution}"
+
+
+class SelectionDiscipline(Enum):
+    FIRST_N = 0
+    RANDOM_N = 1
+    PAD_N = 2
+    EXACT_N = 3
+    K_UNIQUE_N = 4
+
+@dataclass
+class SceneConfig:
+    path: str
+    N: int
```

Comment:
Small suggestion: Can we rename "N" -> e.g. NUM_SCENES ? This is more descriptive